test/N3Writer-test.js

```diff
@@ -569,6 +569,62 @@ describe('Writer', function () {
         done();
       });
     });
+
+    it('should serialize a triple with a triple as subject', function () {
```

Comment:
Make sure to always use different names for your tests, some test runners don't like this.